_news_reader_project/app/src/main/java/com/manu/newsreader/ui/main/model/ArticleItemViewModel.java

```diff
@@ -0,0 +1,25 @@
+package com.manu.newsreader.ui.main.model;
+
+import androidx.lifecycle.ViewModel;
+
+public class ArticleItemViewModel extends ViewModel {
+    public String imagePath;
```

Comment:
It will be somewhat better to have these fields as final (with a constructor) as this way we'll not be able to change them after initial state, less states means less bugs